TelegramBot.py

```diff
@@ -56,13 +57,13 @@ def ping_server(self):
 
 
     def goto_state_running(self):
-        logging.debug("Logging state is turned on running again for " + self.name)
+        logging.debug("Logging state is turned on running again for {}".format(self.name))
```

Comment:
I've just noticed while integrating pylint into the CI that Python's logging functions can do the string interpolation for you:
```py
logging.debug("Logging state is turned on running again for %s", self.name)
```
Unfortunately, they predate the `{}` notation so a C-style `%s` is used instead.